### PR TITLE
Add pyglet renderer to docs.

### PR DESCRIPTION
**Pull request by fywu85**
#275

### CODE REVIEW

File: docs/source/rendering.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+To generate and save the rendering seen above, use
+::
+    exp = minicity_example(render='drgb',  # Render in dynamic RGB colors
+                           save_render=True,  # Save rendering
+                           sight_radius=30,  # Radius of obs.
+                           pxpm=3,  # Render at 3 pixel per meter
+                           show_radius=True)  # Show obs. radius
+
+The rendered frames will be save at ``~/flow_rendering``. For more information,
```
Comment (eugenevinitsky) [inline docs/source/rendering.rst]: nit: will be saved

File: docs/source/rendering.rst [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+.. image:: ../img/minicity_pyglet_render.png
+   :width: 500
+   :align: center
+
+The green arrows are untracked human vehicles, while the blue arrows are RL
+vehicles and tracked human vehicles. Color saturation is proportional to
+speed. For example, the greener the human vehicles, the faster they drive.
+The observation radius of RL and tracked human vehicles are marked by circles.
+
+An example of an extracted local observation is shown as follows.
```
Comment (eugenevinitsky) [inline docs/source/rendering.rst]: nit: is shown below or is as follows

File: docs/source/rendering.rst [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+An example of the minicity rendered by the pyglet renderer can found below.
+
+.. image:: ../img/minicity_pyglet_render.png
+   :width: 500
+   :align: center
+
+The green arrows are untracked human vehicles, while the blue arrows are RL
+vehicles and tracked human vehicles. Color saturation is proportional to
+speed. For example, the greener the human vehicles, the faster they drive.
```
Comment (eugenevinitsky) [inline docs/source/rendering.rst]: nit: they faster they are driving

File: examples/sumo/minicity.py [hunk 4]
```diff
@@ -96,10 +96,10 @@ def minicity_example(render=None,
     # Dynamic grayscale rendering: minicity_example(render="dgray")
     # Static RGB rendering: minicity_example(render="rgb")
     # Dynamic RGB rendering: minicity_example(render="drgb")
-    exp = minicity_example(render=True,
-                           save_render=False,
-                           sight_radius=50,
-                           pxpm=3,
+    exp = minicity_example(render='drgb',
+                           save_render=True,
```
Comment (eugenevinitsky) [inline examples/sumo/minicity.py]: the example should not auto-save the render. This is sort of unpleasant unexpected behavior.

File: docs/source/rendering.rst [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+The observation radius of RL and tracked human vehicles are marked by circles.
+
+An example of an extracted local observation is shown as follows.
+
+.. image:: ../img/local_obs_pyglet_render.png
+   :width: 250
```
Comment (eugenevinitsky) [inline docs/source/rendering.rst]: can you add images of the different rendering modes and options

File: docs/source/index.rst [hunk 6]
```diff
@@ -21,17 +21,25 @@ Flow is a work in progress - input is welcome. Available documentation is limite
...
 
 
 .. Citing Flow
 .. ============
 
-.. If you use Flow for academic research, you are highly encouraged to cite the following paper:
+If you use Flow for academic research, you are highly encouraged to cite our paper:
 
-.. - Yan Duan, Xi Chen, Rein Houthooft, John Schulman, Pieter Abbeel. "`Benchmarking Deep Reinforcement Learning for Continuous Control <http://arxiv.org/abs/1604.06778>`_. *Proceedings of the 33rd International Conference on Machine Learning (ICML), 2016.*
+C. Wu, A. Kreidieh, K. Parvate, E. Vinitsky, A. Bayen, "Flow: Architecture and
```
Comment (eugenevinitsky) [inline docs/source/index.rst]: Thank you!

File: docs/source/index.rst [hunk 7]
```diff
@@ -21,17 +21,25 @@ Flow is a work in progress - input is welcome. Available documentation is limite
    flow_setup
    Tutorials <https://github.com/flow-project/flow/tree/master/tutorials/>
    multiagent
+   rendering
    visualizing
    modules
 
 
 .. Citing Flow
 .. ============
```
Comment (eugenevinitsky) [inline docs/source/index.rst]: Can we uncomment this so it actually shows up?

Comment (fywu85) [reply docs/source/index.rst]: It seems this will be fixed after merging. If not, will create an issue.

**Pull request merged**